spec/html-spec.coffee

```diff
@@ -94,3 +94,12 @@ describe 'HTML grammar', ->
       expect(tokens[2]).toEqual value: '--', scopes: ['text.html.basic', 'comment.block.html', 'invalid.illegal.bad-comments-or-CDATA.html']
       expect(tokens[3]).toEqual value: ' ', scopes: ['text.html.basic', 'comment.block.html']
       expect(tokens[4]).toEqual value: '-->', scopes: ['text.html.basic', 'comment.block.html', 'punctuation.definition.comment.html']
+
+  describe "entities", ->
+    it "tokenizes & and characters after it", ->
+      {tokens} = grammar.tokenizeLine '& &amp; &a'
```

Comment:
@50Wliu I think it is a not yet completed entity. If you have more suggestions just tell me because I don't have experience with this package.